doc: clarify details of TSC public and private meetings (#58925)
nodejs/node: GOVERNANCE.md

The TSC meets in a video conference call. Each year, the TSC elects a chair to
run the meetings. The TSC streams its meetings for public viewing on YouTube.

TSC meetings may consist of a public portion and a private portion. The private
portion is used to discuss sensitive topics, such as personnel issues,
security vulnerabilities, or other confidential matters. Private discussions
should be avoided as much as possible, and the TSC should strive to keep
discussions in the public portion of the meeting, but there are times when

> private discussions are necessary. Any TSC member can request that a topic be
> moved to the private portion of the meeting.

Author: ```suggestion
private discussions are necessary.
```

the TSC. TSC members can also add items to the agenda at the beginning of each
meeting. The meeting chair and the TSC cannot veto or remove items.

The TSC may invite people to take part in a non-voting capacity in either the
public or private portions of the meeting.

During the public portion of the meeting, the TSC chair ensures that someone
takes minutes that include a summary of the discussion and any
decisions made. After the meeting, the TSC chair ensures that someone opens a
public pull request with the minutes from the public portion of the meeting.

The public portion of the TSC meeting is expected to be recorded and made
available for live streaming during the meeting or download by anyone after.
This expectation is to be announced to all participants at the start of the
each meeting before the recording is started. Continued participation in the
public portion of the meeting after this announcement implies consent to the
recording.

> For the private portion of the meeting, the TSC chair ensures that someone
> documents only the final disposition of any decisions made during the meeting
> and ensures that those are shared with all TSC members via a private discussion
> channel such as the TSC private email list. These summaries are not to be made
> public unless all TSC members present during the meeting agree.

Reviewer: ```suggestion
For the private portion of the meeting, the TSC chair ensures that someone
produces a summary of the discussions, gets it reviewed by the attendees
before the meeting ends, and shares it to all the TSC members once approved
by the attendees via a private discussion channel such as the TSC private
mailing list. The summary may be made public if there is consensus within the
TSC to make it public.
```

I think this is closer to what we have already been doing e.g. the chair posts a quick draft of the summary to the Zoom chat, either written by themselves or someone just verbally draft it in the meeting, and ask for objections/feedback on the spot before the meeting ends to post it afterwards.

Author: Re: this part of the suggestion: "The summary may be made public if there is consensus within the TSC to make it public." ...This is not adequate, I think. The agreement needs to be among all the people who were in attendance, not just TSC members.

Author: The only other part here is the "before the meeting ends" ... it is most common that we end up running out of time, participants drop early, etc. It generally hasn't been possible for us to get these summaries reviewed before the end of the call. I don't think we want to try to mandate that they are, just from a practicality perspective.

Reviewer: Dropping `before the meeting ends` SGTM, wasn't intentional to make it limited to be on-spot though I just observed that that's what we've been doing. I guess the chair would just figure it out somehow.

Recording the private portion of a meeting or maintaining or publishing a
detailed transcript is only permitted when all participants present during the
private portion of the meeting explicitly agree to the recording and/or
transcript, in order to comply to privacy regulations.

All discussions made during meetings are considered provisional, receiving no
objections from folks at the TSC meeting to take an action is not equivalent to
the TSC endorsing that action.

If a quorum of TSC voting members is present, it is possible to call for an
explicit vote, and take the vote immediately if there are no objections. The
decision is considered confirmed once the rest of the TSC voting members have
been informed and no objection for taking that vote has been raised in 48 hours.
To clarify, TSC voting members can object to the vote taking place during the
meeting, but not to the vote itself.

> For decisions that are made outside of meetings, the TSC uses

Reviewer: ```suggestion
For discussions outside of meetings, the TSC uses
```

The current wording reads to me that meetings are the primary decision making venue which contradicts with the paragraph above. Also not all discussions lead to decision making, some discussions are just collecting feedback from the TSC or are just informative.

channel such as the TSC private email list. These summaries are not to be made
public unless all TSC members present during the meeting agree.

Recording the private portion of a meeting or maintaining or publishing a
detailed transcript is only permitted when all participants present during the
private portion of the meeting explicitly agree to the recording and/or
transcript. This is to ensure that TSC members and invited guests can feel safe
to discuss sensitive or controversial topics freely without fear of public
scrutiny.

All discussions made during meetings are considered provisional, receiving no
objections from folks at the TSC meeting to take an action is not equivalent to
the TSC endorsing that action.

If a quorum of TSC voting members is present, it is possible to call for an
explicit vote, and take the vote immediately if there are no objections. The
decision is considered confirmed once the rest of the TSC voting members have
been informed and no objection for taking that vote has been raised in 48 hours.
To clarify, TSC voting members can object to the vote taking place during the
meeting, but not to the vote itself.

> For decisions that are made outside of meetings, the uses

Author: ```suggestion
For decisions that are made outside of meetings, the TSC uses
```

available for live streaming during the meeting or download by anyone after.
This expectation is to be announced to all participants at the start of the
each meeting before the recording is started. Continued participation in the
public portion of the meeting after this announcement implies consent to the
recording.

For the private portion of the meeting, the TSC chair ensures that someone
documents only the final disposition of any decisions made during the meeting
and ensures that those are shared with all TSC members via a private discussion
channel such as the TSC private email list. These summaries are not to be made
public unless all TSC members present during the meeting agree.

Recording the private portion of a meeting is forbidden, and no detailed
transcript of the private portion of the meeting is to be maintained or
published. This is to ensure that TSC members and invited guests can feel safe
to discuss sensitive or controversial topics freely without fear of public
scrutiny. Exception is allowed only when all participants present during the
private portion of the meeting explicitly agree to the recording and transcript

> in advance.

Reviewer: I think the suggested wording looks better, and it doesn't make the safety strictly tied to _not recording_, because recording also provides sense of safety too, since being documented discourages harassment.

If a quorum of TSC members is not present, all decisions made during meetings
are considered provisional and subject to confirmation by the full TSC. If no
objections are raised within 48-hours of publishing the summary of a decision

> (either publicly or privately), the decision is considered confirmed.

Author: Oh, much better wording
